jest.config.js

```diff
@@ -2,11 +2,13 @@ export default {
   rootDir: 'web_src',
   setupFilesAfterEnv: ['jest-extended/all'],
   testEnvironment: 'jest-environment-jsdom',
-  testMatch: ['<rootDir>/**/*.test.js'],
+  testMatch: ['<rootDir>/**/*.test.js'], // to run tests with ES6 module, node must run with "--experimental-vm-modules"
   testTimeout: 20000,
   transform: {
     '\\.svg$': '<rootDir>/js/testUtils/jestRawLoader.js',
   },
+  setupFiles: [
+    './js/testUtils/jestSetup.js', // prepare global variables used by our code (eg: window.config)
+  ],
   verbose: false,
 };
-
```

Comment:
No need to drop final newline.

Reply:
But there were 2 new lines before. My change is correct.

<img width="447" alt="image" src="https://user-images.githubusercontent.com/2114189/194698579-058e04d5-0d50-40f4-9e54-c57988642dd1.png">

Reply:
```
curl --proxy http://127.0.0.1:1080/  https://raw.githubusercontent.com/go-gitea/gitea/main/jest.config.js | hexdump
...
0000120 3b7d 0a0a
```

Reply:
Huh, you are right. GitHub UI completely hides this fact that there are two newlines, but I see it when I view the file locally.